src/rust-crypto/backup.ts

```diff
@@ -323,7 +324,14 @@ export class RustBackupManager extends TypedEventEmitter<RustBackupCryptoEvents,
         await sleep(delay);
 
         try {
-            let numFailures = 0; // number of consecutive network failures for exponential backoff
+            // number of consecutive network failures for exponential backoff
+            let numFailures = 0;
+            // `olmMachine.roomKeyCounts()` is very slow on some configurations (see more comments below),
+            // we compute it only once per `backupKeysLoop` if necessary.
```

Comment:
I feel like it's more important to document here what value the variable actually holds, rather than why it exists. 

```suggestion
            // The number of keys left to back up. (Populated lazily: see more comments below.)
```